device/std.self.json

```diff
@@ -0,0 +1,28 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azure:common:SelfDescribing;1",
```

Comment:
Any suggestion to define the ID for this interface?

Reply:
So we're taking feedback on the model Id, but not the rest of the model?

Reply:
I'm asking about the model is because it has not been fully decided, if you have any other feedback about the model, please share it.

Reply:
I did give feedback about the model for the properties, and you told me to pound sand, no?